Dockerfile.std

```diff
@@ -1,16 +1,21 @@
 # Generated by: tyk-ci/wf-gen
-# Generated on: Thu 29 Apr 11:02:17 UTC 2021
+# Generated on: Mon 10 May 06:43:08 UTC 2021
 
 # Generation commands:
-# ./pr.zsh -title custom goreleaser -branch releng/arm64 -base releng/arm64 -p
+# ./pr.zsh -title docker builds are parallel -repos tyk -branch releng/python -base releng/python -p
 # m4 -E -DxREPO=tyk
 
 FROM debian:buster-slim
 ARG TARGETARCH
 
 RUN apt-get update \
  && apt-get dist-upgrade -y ca-certificates \
- && apt-get autoremove -y
+  python3-setuptools libpython3.7 curl \
```

Comment:
I believe you'll also need `python3-dev` in order to get the pkgconfig for it, on which `dlpython` relies to find the runtime. IIRC it also pulls a lot of unneeded dependencies for development, so the old `Dockerfile` was cleaning this up additionally as otherwise the image grows pretty large:
https://github.com/TykTechnologies/tyk-gateway-docker/blob/d7fca828207b784f6561dc36f44a61e490670fa0/Dockerfile#L20-L21

Same for the hybrid image.

Unless it now works differently and not needed?

Reply:
Fixed in last commit on this branch.